api/v1alpha2/loadbalancer/types.go

```diff
@@ -0,0 +1,111 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package loadbalancer
+
+import (
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	// AwsProvider is the name of the aws provider
+	AwsProvider = "aws"
```

Comment:
I think the provider name should be explicit to the LB implementation, not the cloud provider, so this should be "ELB", thoughts?

Reply:
actually this should be removed. As the current PR uses `Kind` to store/retrieve the provider